apps/extension/src/state/tx-approval.ts

```diff
@@ -1,13 +1,13 @@
+import type { TransactionView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1alpha1/transaction_pb';
+import type { AuthorizeRequest } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/custody/v1alpha1/custody_pb';
+import type { Stringified } from '@penumbra-zone/types';
 import { AllSlices, SliceCreator } from './index';
-import { AuthorizeRequest } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/custody/v1alpha1/custody_pb';
-import { Jsonified } from '@penumbra-zone/types';
 import { MessageResponder } from '@penumbra-zone/types/src/internal-msg/shared';
-import { TransactionView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1alpha1/transaction_pb';
 import { TxApproval } from '@penumbra-zone/types/src/internal-msg/tx-approval';
 
 export interface TxApprovalSlice {
-  authorizeRequest?: Jsonified<AuthorizeRequest>;
-  transactionViewFromPlan?: Jsonified<TransactionView>;
+  authorizeRequest?: Stringified<AuthorizeRequest>;
+  transactionViewFromPlan?: Stringified<TransactionView>;
```

Comment:
Is this because of some zustand/immer type thing why we can't use JsonValue? Regardless, can you add a code comment above talking about why we specifically stringify?